src/PockyBot.NET.Persistence/Models/Peg.cs

```diff
@@ -0,0 +1,25 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace PockyBot.NET.Persistence.Models
+{
+    [Table("pegs")]
+    public class Peg
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Column("id")]
+        public int Id { get; set; }
+        [Column("sender")]
+        public string Sender { get; set; }
```

Comment:
um... that is what's happening here. Sender and Receiver refer to the user ID, and EF is able to use that to get the user objects because they're foreign keys. So that's where "SenderUser" and "ReceiverUser" come from (they don't actually exist on the table, but we can get to them)